web/src/components/DandisetsPage.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div v-page-title="pageTitle">
```

Comment:
I guess this fixes a bug with the page title?

Reply:
Yeah, I described it in https://github.com/dandi/dandi-archive/pull/2238/commits/74452184f4a4ad714959ecbfb51073f7cfc3155e